src/components/CharacterInfoField/CharacterInfoField.css

```diff
@@ -0,0 +1,58 @@
+:host {
+  border: 1px solid var(--background-color-heading);
+
+  display: flex;
+  align-items: center;
+}
+:host(:focus-within) {
+  box-shadow: 0 0 10px var(--color-shadow-primary);
+}
+label {
+  font-family: "Signika Negative", "Signika", sans-serif;
+  font-size: var(--spacing-full); /* TODO: variables! ems! */
+  line-height: 38px !important;
```

Comment:
We only get to use Foundry's font scaling settings if we use Foundry's own font size variables